runner/android_junit_runner/java/androidx/test/filters/MediumTest.java

```diff
@@ -22,23 +22,23 @@
 import java.lang.annotation.Target;
 
 /**
- * Annotation to assign a medium test size qualifier to a test. This annotation can be used at a
+ * Annotation to assign a medium test size qualifier to a test. This annotation can be used at the
  * method or class level.
  *
  * <p>Test size qualifiers are a great way to structure test code and are used to assign a test to a
- * test suite of similar run time.
+ * test suite or similar runtime.
```

Comment:
In my understanding the goal of this annotation is that all "Medium" sized test will be in an implicit "test suite", so that all similarly long tests can be executed at once. So the test suite wouldn't have similar run times, the other tests in the suite would have similar run times to the one that's annotated.

I think the original sentence has a missing implicit ending that's why it's harder to understand:

```diff
-used to assign a test to a test suite of similar run time.
+used to assign a test to a test suite of similar run timed tests.
```

Reply:
I see. How about "used to assign a test to a suite of tests that have similar run times"?